MAUI/Autocomplete/Getting-Started.md

```diff
@@ -0,0 +1,235 @@
+        this.SocialMedias.Add(new SocialMedia() { Name = "Google Plus", ID = 1 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Instagram", ID = 2 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "LinkedIn", ID = 3 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Skype", ID = 4 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Telegram", ID = 5 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Televzr", ID = 6 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tik Tok", ID = 7 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tout", ID = 8 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Tumblr", ID = 9 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Twitter", ID = 10 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "Vimeo", ID = 11 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "WhatsApp", ID = 12 });
+        this.SocialMedias.Add(new SocialMedia() { Name = "YouTube", ID = 13 });
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+**Step 2:** Populate data in `Autocomplete`. 
+
+Now, populate this 'SocialMediaViewModel' data in the `Autocomplete` control by binding to the `ItemsSource` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:editors="clr-namespace:Syncfusion.Maui.Inputs;assembly=Syncfusion.Maui.Inputs"
+             xmlns:local="clr-namespace:AutocompleteSample"             
+             x:Class="AutocompleteSample.MainPage">
+
+       <ContentPage.BindingContext>
+            <local:SocialMediaViewModel />
+       </ContentPage.BindingContext>
+
+       <ContentPage.Content>
+            <!--Setting ItemsSource-->
+            <editors:SfAutocomplete x:Name="autocomplete" 
+                                    Width="250"
+                                    ItemsSource="{Binding SocialMedias}" />
+        </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
+autocomplete.BindingContext = socialMediaViewModel;
+autocomplete.ItemsSource = SocialMediaViewModel.SocialMedias;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Set the 'SocialMediaViewModel' instance as the `BindingContext` of your control; this is done to bind properties of 'SocialMediaViewModel' to `Autocomplete`.
+
+**Step 3:** Set the `TextMemberPath` and `DisplayMemberPath`.
+
+The `Autocomplete` control is populated with the list of social media. However, because the 'SocialMedia' model contains two properties 'Name' and 'ID', it is necessary to intimate, which property should be a display value in the selection box portion and drop-down suggestion of the `Autocomplete` control.
```

Comment:
The `Autocomplete` control is populated with the list of social media. However, because the 'SocialMedia' model contains two properties 'Name' and 'ID', it is necessary to intimate which property should be a display value in the selection box portion and drop-down suggestion of the `Autocomplete` control.

Reply:
Changes are made.